### ISSUE TITLE
Inline widget in readonly mode is possible to remove with draghandler

### ISSUE THREAD

**Issue by msamsel** [ckeditor/ckeditor-dev#1657]
## Are you reporting a feature request or a bug?

Bug

## [Check if the issue is already reported](https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_issues_readme-section-avoid-duplicates)

follow up after #1570 

## Provide detailed reproduction steps (if any)

1. Visit http://cdn.ckeditor.com/4.8.0/full-all/samples/old/image2/image2.html
2. Transform image2 to inline widget by removing its caption and removing alignment in image properties.
3. Set editor as readonly `CKEDITOR.instances.editor1.setReadOnly( true )`.
4. Use draghandler to move widget around (e.g. above header )

[Feb-12-2018 16-50-03.mp4.zip](https://github.com/ckeditor/ckeditor-dev/files/1716895/Feb-12-2018.16-50-03.mp4.zip)

### Expected result

widget disappear

### Actual result

Widget should remain in editor

## Other details

* Browser: …
* OS: …
* CKEditor version: …
* Installed CKEditor plugins: …

**Comment by engineering-this** [comment]
Duplicate of #808
